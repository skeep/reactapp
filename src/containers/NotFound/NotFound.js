import React from 'react'
import { Helmet } from 'react-helmet'

const NotFound = () => (
  <div>
    <Helmet>
      <title>Not Found 404</title>
    </Helmet>
    <h2>Not Found</h2>
  </div>
)

export default NotFound
