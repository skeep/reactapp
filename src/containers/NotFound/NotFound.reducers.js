const NotFoundState = (state = {}, action) => {
  switch (action.type) {
    default:
      return state
  }
}

export default NotFoundState
