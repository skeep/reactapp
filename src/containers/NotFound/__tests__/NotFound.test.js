import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import NotFound from '../NotFound.js'

// Snapshot test
it('Snapshot test with content', () => {
  const tree = shallow(<NotFound />)
  expect(toJson(tree)).toMatchSnapshot()
})

// Rendering test
it('Rendering test with content', () => {
  const tree = shallow(<NotFound />)
  expect(tree.find('h2').text()).toEqual('Not Found')
})

// Behaviour tests

// Integration tests

// System tests
