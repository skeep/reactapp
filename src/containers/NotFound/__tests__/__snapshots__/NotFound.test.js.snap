// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test with content 1`] = `
<div>
  <h2>
    Not Found
  </h2>
</div>
`;
